src/main/java/com/aws/greengrass/device/ClientDevicesAuthService.java

```diff
@@ -155,6 +160,11 @@ private int getValidCloudCallQueueSize(Topics topics) {
     /**
      * Certificate Manager Service uses the following topic structure:
      * |---- configuration
+     * |    |---- settings:
+     * |         |---- cloudQueueSize: "..."
+     * |         |---- threadPoolSize: "..."
+     * |         |---- deviceAuthToken:
```

Comment:
I don't think this necessarily needs to be its own namespace. maxActiveAuthTokens fits fine IMO next to the other options.

Reply:
moved under "performance"